src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -204,7 +201,7 @@ class AddressPage extends Component {
                         <CountryPicker
                             inputID="country"
                             onValueChange={this.onCountryUpdate}
-                            defaultValue={address.country || ''}
+                            defaultValue={countryISOcode || ''}
```

Comment:
I think we can inline this yeah?

```suggestion
                            defaultValue={PersonalDetails.getCountryISO(address.country)}
```